source/Sashimi.Azure.Accounts/AccountTypes.cs

```diff
@@ -5,5 +5,6 @@ namespace Sashimi.Azure.Accounts
     public static class AccountTypes
     {
         public static readonly AccountType AzureServicePrincipalAccountType = new AccountType("AzureServicePrincipal");
+        public static readonly AccountType AzureSubscriptionAccountType = new AccountType("AzureSubscription");
```

Comment:
I needed to copy the account type from AzureCloudService. We can update AzureCloudService to refer to this

Reply:
This was in CloudServices on purpose. CS is the only thing that should know that account type exists, we don't want anyone else using it and if/when CS goes away this account type should go with it.

Reply:
Then should we make this internal, since we need it for https://github.com/OctopusDeploy/Sashimi/pull/90/files#diff-fa199ea3483005f7ffd6bf84edab18edR18

Reply:
Reverted